packages/docs/src/components/HomepageFeatures.tsx

```diff
@@ -32,7 +32,7 @@ const FeatureList = [
   },
 ];
 
-function Feature({Svg, title, description}) {
```

Comment:
suggestion: if you're going to convert to tsx then add the types, otherwise just leave it as js for the time being